.github/workflows/docker-arm.yml

```
@@ -0,0 +1,46 @@
+name: Docker-ARM
+
+on: [push, pull_request]
+
+env:
+  IMAGE_NAME: wine
+  DOCKER_USERNAME: zixia
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+      # This is used to complete the identity challenge
+      # with sigstore/fulcio when running outside of PRs.
+      id-token: write
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Docker Setup QEMU
+        uses: docker/setup-qemu-action@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Log into registry
+        uses: docker/login-action@v3
+        with:
+          username: ${{ env.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PERSONAL_ACCESS_TOKEN }}
+
+
+      - name: Build and push Docker image
+        id: build-and-push
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          file: ./Dockerfile.arm
+          platforms: |-
+            linux/arm64
+          push: true
```

```suggestion
          push: ${{ github.event_name != 'pull_request' }}
```

I guess you don't want to push every PR to docker. Should also help with the missing secrets if you add a similar `if` above to the login step.